test/examples/ucp_client_server.c

```diff
@@ -737,7 +776,8 @@ int main(int argc, char **argv)
     ucp_context_h ucp_context;
     ucp_worker_h  ucp_worker;
 
-    ret = parse_cmd(argc, argv, &server_addr, &listen_addr, &send_recv_type);
+    ret = parse_cmd(argc, argv, &server_addr, &listen_addr, &send_recv_type,
+                    &client_server_iterations);
```

Comment:
why need to pass &client_server_iterations, it's global var

Reply:
oh it's a leftover.. will remove everywhere